media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+.firefox-friends {
+    #wrapper {
+        padding-bottom: 0;
+        width: 100%;
+    }
+    #masthead {
+        background: #373737 url(/media/img/contribute/friends/header-img-wide.jpg) 50% 0 no-repeat;
+        padding: 0 180px;
+        width: 100%;
+        .border-box();
+
+        @media only screen and (max-width: 1100px) {
+            padding: 0 100px;
+        }
+
+        @media only screen and (max-width: 790px) {
+            padding: 0 50px;
+        }
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            padding: 0 20px;
+        }
+    }
+    #colophon {
+        margin-top: 0;
+    }
+    h1,
+    h2,
+    h3 {
+        color: #fff;
+        font-weight: 700;
+    }
+    h1 {
+        .font-size(4rem);
+        text-shadow: 1px 1px 2px #333;
+
+        @media only screen and (max-width: 790px) {
+            .font-size(3rem);
+        }
+    }
+    h2 {
+        .font-size(2.5rem);
+    }
+    h3 {
+        .font-size(1.2rem);
+        text-transform: uppercase;
+
+        @media only screen and (max-width: 790px) {
+            .font-size(1rem);
+            line-height: 1.5;
+        }
+    }
+    hr {
+        margin: 100px auto;
+        border: 1px solid #fff;
+        width: 20%;
+    }
+    .ff-logo {
+        padding-top: 25px;
+    }
+    .ff-header-content {
+        color: #fff;
+        margin: 0 auto;
+        padding-top: 95px;
+        padding-bottom: 160px;
+        width: 100%;
+        text-align: center;
+
+        p {
+            .font-size(1.4rem);
+            text-shadow: 1px 1px 2px #333;
+
+            @media only screen and (max-width: 790px) {
+                .font-size(1.2rem);
+            }
+        }
+    }
+    .ff-join-now {
+        display: block;
+        background-color: #f26c24;
+        color: #fff;
+        margin: 70px auto 0 auto;
+        padding: 10px 30px;
+        border: 0;
+        width: 200px;
+        .font-size(1.2rem);
+        .border-radius(6px);
```

Comment:
Nit: no need to use the mixin for border-radius as unprefixed support is good enough now :)